doc/tutorial/statistical_inference/model_selection.rst

```diff
@@ -70,7 +70,7 @@ This example shows an example usage of the ``split`` method.
 
 The cross-validation can then be performed easily::
 
-    >>> kfold = KFold(n_splits=3)
+    >>> k_fold = KFold(n_splits=3)
```

Comment:
I removed the line and I'll wait for the CI. I'll merge this one they come back green.